fix window lib path for renv; updates to work with new data flow

youyifong opened the pull request:
…ork with new data flow

benkeser commented on cor_nonlinear/code/cor_nonlinear.R:
```diff
@@ -1,55 +1,70 @@
+    options(renv.config.install.transactional = FALSE)
+    renv::restore(library=saved.system.libPaths, prompt=FALSE) # for a quick test, add: packages="backports"
+    .libPaths(c(saved.system.libPaths, .libPaths()))
+} else renv::restore(prompt=FALSE)
```
@nhejazi What's the recommendation for this? `restore` or `activate`?

I've been doing manual `restore()`s from time to time.

nhejazi replied:
It would be better to avoid automatically calling `renv::restore()`

nhejazi commented:
@youyifong --- would you mind creating a new PR that includes only the relevant changes to `cor_coxph` and `cor_nonpar`? I'm going to close this due to conflicts with the `make_dat_proc.R` script. If those changes are still necessary, please create a separate PR (that does not include any changes to `cor_coxph` and `cor_nonpar`) and re-submit.

youyifong commented:
Hi Nima, in PR #1, I removed the lines related to restore in
make_dat_proc.R that I added.
Best,
Youyi

On Wed, Mar 3, 2021 at 2:28 PM Nima Hejazi <notifications@github.com> wrote:

> @youyifong <https://github.com/youyifong> --- would you mind creating a
> new PR that includes only the relevant changes to cor_coxph and cor_nonpar?
> I'm going to close this due to conflicts with the make_dat_proc.R script.
> If those changes are still necessary, please create a separate PR (that
> does not include any changes to cor_coxph and cor_nonpar) and re-submit.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CoVPN/correlates_reporting/pull/91#issuecomment-790118215>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHBH5LY3BO5MK76PXU2UHM3TB2ZZBANCNFSM4YOH233Q>
> .
>
